doc/source/admin/cluster.md

```diff
@@ -314,6 +315,63 @@ Torque attributes can be defined in either their short (e.g. [qsub(1B)](http://c
 
 Most options available to `qsub(1b)` and `pbs_submit(3b)` are supported.  Exceptions include `-o/Output_Path`, `-e/Error_Path`, and `-N/Job_Name` since these PBS job attributes are set by Galaxy.
 
+## AWS Batch
+
+Runs jobs via the [AWS Batch](https://aws.amazon.com/batch/). Built on top of AWS Elastic Container Service (ECS), AWS Batch enables users to run hundreds of thousands of jobs with simple configuration.
+
+#### Dependencies
+
+AWS Batch job runner requirs AWS Elastic File System (EFS) being mounted as a shared file system that enables Galaxy and job containers to read and write files. In the best pratice, Galaxy is installed an AWS EC2 instance and an EFS is mounted to the EC2 as a local drive. Job-related paths, such as objects, jobs_directory, tool_directory and so on, need to be placed on the EFS drive.
```

Comment:
```suggestion
The AWS Batch job runner requires an AWS Elastic File System (EFS) to be mounted as a shared file system that enables Galaxy and job containers to read and write files. Galaxy is first installed on an AWS EC2 instance with EFS mounted as a local drive. Job-related paths, such as objects, jobs_directory, tool_directory and so on, need to be placed on the EFS drive.
```

Reply:
@nuwang Thanks for your great help on my English. For `Galaxy is first installed on an AWS EC2 instance with EFS mounted as a local drive`, my understanding is that installation of Galaxy on EC2 is not necessary once EFS is able to be mounted. The AWS direct connect seems to allow mounting of EFS to happen outside of AWS, although no tests have been done yet.

Reply:
Sure, makes sense. Please reword if you think it's appropriate.